Bugzilla/API/V1/Configuration.pm

```diff
@@ -0,0 +1,273 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::API::V1::Configuration;
+
+use 5.10.1;
+
+use Bugzilla::Constants;
+use Bugzilla::Field;
+use Bugzilla::Keyword;
+use Bugzilla::Logging;
+use Bugzilla::Product;
+use Bugzilla::Status;
+
+use List::MoreUtils qw(any);
+use Mojo::Base qw( Mojolicious::Controller );
+use Mojo::JSON qw( true false );
+
+sub setup_routes {
+  my ($class, $r) = @_;
+  $r->get('/latest/configuration')->to('V1::Configuration#configuration');
```

Comment:
If we're providing backwards compat we should also hook up routes for `/rest/configuration` and `/bzapi/configuration`, even if they just redirect to the canonical `/latest/configuration` endpoint.

edit: ah, I see you're defining `/rest/configuration` in CGI.pm and `/bzapi/configuration` in the BzAPI extension.  Both of those should be moved here.